pkg/universe.dagger.io/alpha/terraform/run.cue

```diff
@@ -64,11 +68,12 @@ _#DefaultLogLevel: "off"
 
 	// Hashicorp Terraform container
 	container: #input: docker.#Image | #Image
-
 	// Run command within a container
 	_run: docker.#Build & {
 		steps: [
-			container.#input,
+			container.#input & {
+				"version": version
```

Comment:
IIUC `container.#input` can be either a `docker.#Image` or `#Image`. 

If a `docker.#Image` is used how come this conjunction could be possible? By looking at this, I'm not sure if the `docker.#Image` flow will kept working after this PR

Reply:
@marcosnils Good catch! I have moved the input for version to the #Image definition. Please let me know if this resolves your concerns.